bundler/lib/bundler/cli.rb

```diff
@@ -620,7 +620,7 @@ def self.source_root
     method_option "dry-run", :type => :boolean, :default => false, :banner =>
       "Only print out changes, do not clean gems"
     method_option "force", :type => :boolean, :default => false, :banner =>
-      "Forces clean even if --path is not set"
+      "Forces clean even if system location is used"
```

Comment:
I think "system location" may still not be 100% clear. Maybe we could elaborate a bit more, like

> Forces cleaning up unused gems even if Bundler is configured to install and look for gems globally, and as a consequence, all system gems except for the one in the current application would be removed.

Perhaps other maintainers can propose a better wording that explains this more concisely.

Reply:
@syohex Thanks for proposing this even if not familiar with Bundler. I tweaked my suggested warning a bit, can you update this PR to use that?

Reply:
@deivid-rodriguez I have updated at 7c54dc56f651f93d37f8b6329e893211d5ca2e9b